README.md

```diff
@@ -256,20 +256,24 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600,
 
-                // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "irods_user_claim": "irods_username",
-
-                // The mapping of a user to the provided values. All values must
-                // be matched to map an OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "user_attribute_mapping": {
-                    "irods_username": {
-                        "sub": "123-abc-456-xyz",
-                        "email": "rods_user@example.org"
+                // User mapping plugins replace 'user_attribute_mapping' and
+                // 'irods_user_claim'. Addtional plugins may be developed to
+                // suit specific needs.
+                "user_mapping": {
+                    // Full path to plugin to load.
+                    // Currently we supply the following plugins:
+                    //   - local_file
+                    //   - user_claim
+                    "plugin_path": "/usr/lib/irods_http_api/plugins/user_mapping/libirods_http_plugin-local_file.so",
```

Comment:
Do we want to list available plugins?

Do we want the example to have a full path?

Reply:
i think this is fine for now... we'll learn more as anyone sees it and tries to figure out what's happening.

Reply:
Agreed.

What you have is good.

The introduction of plugins does mean users can't run multiple versions of the HTTP API on the host computer (because of packaging). I don't necessarily see that as a problem though. No will want to run multiple versions on the same computer. But if they need to, they can use docker.

Reply:
Gotcha, resolving then.